docs/styles/style.css

```diff
@@ -70,16 +107,135 @@ h5 {
   font-size: calc(var(--typo-scale) * 1rem);
 }
 
+@media screen and (min-width: 481px) {
+  .bio h5 {
+    margin-top: 0;
+  }
+}
+
+time {
+  border-bottom: 1px solid #dfdfdf;
+  display: block;
+  margin-bottom: 2rem;
+  padding-bottom: 0.2rem;
+  font-size: 0.9rem;
+  font-family: "Poppins", Arial, sans-serif;
+  color: #646464;
+}
+
 /* LAYOUT */
 
-.bio {
+header img {
+  object-fit: cover;
+  width: 100%;
+  height: 10rem;
+  margin-bottom: 2.5em;
+}
+
+@media screen and (min-width: 481px) {
+  header img {
+    height: 15rem;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  header img {
+    height: 20rem;
+  }
+}
+
+.meta-tag a {
+  box-shadow: none;
+  color: #333333;
+  font-size: 0.9rem;
+  background-color: #e8eaec;
+  border-radius: 5px;
+  padding: 0.05rem 0.6rem 0.2rem 0.6rem;
+}
+
+.meta-tag a:hover {
+  background-color: #5aabf4;
+}
+
+.article-col-container {
+  display: grid;
+  grid-template-rows: auto auto;
+}
+
+.article-col-container__col ol, .article-col-container__col ul {
+  margin-top: 0;
+}
+
+@media screen and (min-width: 481px) {
+  .article-col-container {
+    grid-template-rows: auto auto;
+  }
+  .article-col-container__col ol, .article-col-container__col ul {
+    margin-top: 0;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .article-col-container {
+    grid-template-columns: 50% 50%;
+  }
+
+  .article-col-container__col ol, .article-col-container__col ul {
+    margin-top: 0;
+  }
+}
+
+article, .bio, footer {
+  width: 90%;
+  margin: 0 auto 2rem auto;
+}
+
+@media screen and (min-width: 481px) {
+  article, .bio, footer {
+    width: 70%;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  article, .bio, footer {
+    max-width: 60rem;
```

Comment:
You can probably have that already in the default definition and not only in the media query. You would probably never have a wider column than e certain amount (50–80 is a good indicator) of characters. But of course it never kicks in before the wide screens but saves you one additional media-query and selector.